docs/html-css/pre-work/challenge.md

```diff
@@ -0,0 +1,17 @@
+The Challenge is to create a responsive portfolio website using HTML, CSS.
+
+**Add Different Pages including:-**
+
+-Home Page
+Add user's image and name
+
+-About Page
+Information about the user such as interest, hobbies, technical experience
+
+-Skill Page
+List all the languages and frameworks the user is familiar with
+
+-Contact Page
+List all the social media links such as GitHub, LinkedIn, and so on.
+
+**The requirement is that the website should be responsive and have a decent theme with a regular color combination**
```

Comment:
Students won't have learnt about Responsive Design yet since this is before the HTML module

Reply:
Ok, I will update it.